doc/source/tokudb/tokudb_troubleshooting.rst

```diff
@@ -1254,10 +1254,10 @@ The following is a reference of table status statements:
  Number of failed calls to ``realloc()``.
 
 ``memory: number of bytes requested``:
- Total number of bytes requested from mallocator.
+ Total number of bytes requested from memory allocator library.
 
 ``memory: number of bytes freed``:
- Total number of mallocated bytes freed (used - freed = bytes in use).
+ Total number of memory allocated bytes freed (used - freed = bytes in use).
```

Comment:
Missed one, maybe "Total number of bytes allocated from memory allocation library that have been freed (used - freed = bytes in use)."